### PR TITLE
fix #35573 divamagazin.com

### LINKED ISSUE #35573: divamagazin.com
### Issue URL (Ads)
[https://divamagazin.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fdivamagazin.com%2F)
### Comment
Username: @PentagramTr
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.1.3
Browser: | Firefox
Stealth mode options: | Hide your search queries, Send Do-Not-Track header, Strip URLs from tracking parameters, Self-destructing third-party cookies (2880), Block WebRTC, Hide your Referrer from third-parties
Filters: | AdGuard Turkish,<br>AdGuard Annoyances,<br>AdGuard Base,<br>NoCoin Filter List,<br>AdGuard Tracking Protection,<br>AdGuard Social Media

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/35573

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -23,6 +23,8 @@ gamereactor.de##.socialIcons
 gamereactor.de###facebooknotification
 daenischesbettenlager.at,daenischesbettenlager.de,jysk.al,jysk.am,jysk.ba,jysk.be,jysk.bg,jysk.by,jysk.ca,jysk.co.id,jysk.co.uk,jysk.com,jysk.com.cy,jysk.com.kw,jysk.com.mt,jysk.ch,jysk.cz,jysk.dk,jysk.ee,jysk.es,jysk.fi,jysk.fr,jysk.ge,jysk.gr,jysk.hr,jysk.hu,jysk.ie,jysk.it,jysk.kz,jysk.lt,jysk.lv,jysk.nl,jysk.md,jysk.me,jysk.mk,jysk.no,jysk.pl,jysk.pt,jysk.ro,jysk.rs,jysk.se,jysk.si,jysk.sk,jysk.ua,jysk.vn,jysk-ks.com,jyskthailand.com,pisiffik.gl,rumfatalagerinn.is,skemman.fo###social-share
 doeda.com##.entry-actions
+divamagazin.com###insta-celeb
+divamagazin.com###vitrin_ile_astro
```
Comment (Alex-302) [inline SocialFilter/sections/specific.txt]: looks like an ad

File: SocialFilter/sections/specific.txt [hunk 2]
```diff
@@ -23,6 +23,8 @@ gamereactor.de##.socialIcons
 gamereactor.de###facebooknotification
 daenischesbettenlager.at,daenischesbettenlager.de,jysk.al,jysk.am,jysk.ba,jysk.be,jysk.bg,jysk.by,jysk.ca,jysk.co.id,jysk.co.uk,jysk.com,jysk.com.cy,jysk.com.kw,jysk.com.mt,jysk.ch,jysk.cz,jysk.dk,jysk.ee,jysk.es,jysk.fi,jysk.fr,jysk.ge,jysk.gr,jysk.hr,jysk.hu,jysk.ie,jysk.it,jysk.kz,jysk.lt,jysk.lv,jysk.nl,jysk.md,jysk.me,jysk.mk,jysk.no,jysk.pl,jysk.pt,jysk.ro,jysk.rs,jysk.se,jysk.si,jysk.sk,jysk.ua,jysk.vn,jysk-ks.com,jyskthailand.com,pisiffik.gl,rumfatalagerinn.is,skemman.fo###social-share
 doeda.com##.entry-actions
+divamagazin.com###insta-celeb
```
Comment (Alex-302) [inline SocialFilter/sections/specific.txt]: block with script(domain specific)

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Comment by PentagramTr** [comment]
`###insta-celeb`

This is the site's Instagram page.I think it should not be blocked.

https://kb.adguard.com/en/general/adguard-filter-policy#limitations-and-exceptions-2

```
Links to website’s communities in social networks are not blocked.
```

**Comment by PentagramTr** [comment]
@zloyden 

I think you forgot this ad.Please check it.

<details><summary>Screenshot:</summary>

divamagazin com

</details>

**Comment by zloyden** [comment]
> @zloyden
> 
> I think you forgot this ad.Please check it.
> 
> Screenshot:

It wasn't showing on my side before. Thanks!

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**